frontend/catalyst/jit.py

```diff
@@ -279,6 +279,41 @@ def f(x):
         Under the hood, Catalyst converts anything coming in the latter notation into the
         former one.
 
+        Similarly, to update array values with an operation when using JAX, the JAX syntax for array
```

Comment:
Good idea to also update the `qjit` docstring, although it is getting quite long. Maybe you can find a way to combine this example with one above to save some space?